src/runtime/cmd/kata-runtime/kata-env.go

```diff
@@ -123,6 +123,8 @@ type HypervisorInfo struct {
 	MemorySlots       uint32
 	HotPlugVFIO       config.PCIePort
 	ColdPlugVFIO      config.PCIePort
+	PCIeRootPort      uint32
```

Comment:
Slight typo in the cover letter, `continerd` instead of `containerd` (twice)
<img width="484" alt="image" src="https://github.com/kata-containers/kata-containers/assets/1695924/a4b6a064-861c-40bc-bc01-00903ddd7445">